src/main/drivers/timer.c

```diff
@@ -655,6 +655,12 @@ _TIM_IRQ_HANDLER(TIM4_IRQHandler, 4);
 #if USED_TIMERS & TIM_N(5)
 _TIM_IRQ_HANDLER(TIM5_IRQHandler, 5);
 #endif
+#if USED_TIMERS & TIM_N(6)
+_TIM_IRQ_HANDLER(TIM6_IRQHandler, 6);
```

Comment:
It may be possible to define some macro magic to validate IRQ handlers, something like 
`IRQ_HANDLER(TIM6_DAC_IRQ)`, forcing it to check that TIM6_DAC_IRQn is defined (and has numeric value)